README.md

```diff
@@ -36,7 +36,7 @@ If you have a Node and Go environment you can also build the production version
 
 ```
 npm install
-NODE_ENV=production webpack -p --progress
+npm run-script build
```

Comment:
It is mentioned here. Maybe there is a better fix to make npm build work as well.